source/FreeRTOS_Sockets.c

```diff
@@ -2900,7 +2900,14 @@ BaseType_t FreeRTOS_setsockopt( Socket_t xSocket,
                         /* More details at: https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/MISRA.md#rule-111 */
                         /* coverity[misra_c_2012_rule_11_8_violation] */
```

Comment:
Can we please move these 2 lines inside ifdef, the MISRA suppression should be immediately followed by the MISRA issue.